aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -58,6 +58,7 @@ class AztecToolbar : FrameLayout, IAztecToolbar, OnMenuItemClickListener {
     private lateinit var layoutExpandedTranslateInEnd: Animation
     private lateinit var layoutExpandedTranslateOutStart: Animation
 
+    private lateinit var HTMLButton: RippleToggleButton
```

Comment:
This is just a nitpick, but we've been using lowercase for acronyms in variable names. So for consistency I'd name this `htmlButton`. Feel free to ignore though :).

Reply:
No nitpick at all, renamed it!